### DISCUSSION TITLE
plugins : topology view customization

### DISCUSSION THREAD

**Discussion by othmaneSH24** [netbox-community/netbox#8939] [General]
hello everyone ,  

For my project i need to customize the topology view  in the plugins , my purpose is to have an icon representation of each devices in the topology view , however i can't find in Netbox where can i do this customization .

Thank you in advance for your help .

**Comment by DanSheps** [comment]
What plugin are you using?  You should likely ask the plugin author.

**Reply by othmaneSH24** [reply]
I am using  0.9.2 version , I  need to assign a tag for every devices so i can have a vison of some icons  in the topology view  , do you have an idea on how we do that ?

**Reply by emersonfelipesp** [reply]
@othmaneSH24 you probably is using **[netbox-ui-plugin](https://github.com/iDebugAll/nextbox-ui-plugin)** or **[netbox-topology-views](https://github.com/mattieserver/netbox-topology-views)**.
You must go there to find any help for your problem, because Netbox does not support official support for any plugins, as I know.
